Sources/BranchSDK/BNCEncodingUtils.m

```diff
@@ -177,38 +176,30 @@ + (NSString *)encodeDictionaryToJsonString:(NSDictionary *)dictionary {
         id obj = dictionary[key];
         if ([obj isKindOfClass:[NSString class]]) {
             value = [BNCEncodingUtils sanitizedStringFromString:obj];
-        }
-        else if ([obj isKindOfClass:[NSURL class]]) {
+        } else if ([obj isKindOfClass:[NSURL class]]) {
```

Comment:
Nit: Personally i prefer if/else statements beginning on new lines since the collapse buttons on certain IDEs pushes the subsequent statement to the end of the line of the collapsed function. (not blocking) 

<img width="655" alt="Screenshot 2024-04-01 at 7 18 33 PM" src="https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/assets/94577415/b7d85ba9-67e4-47d8-9943-90270e17c3cd">
<img width="877" alt="Screenshot 2024-04-01 at 7 18 41 PM" src="https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/assets/94577415/ba8f625a-88ba-4766-a9fd-846ac3e20a0f">

Reply:
The Objective-C style guide is to put it on the same line. Similar to K&R and Linux kernel style. https://google.github.io/styleguide/objcguide.html#conditionals

Reply:
Xcode also doesn't have that neat feature to fold code nicely.